src/aioquic/quic/connection.py

```diff
@@ -398,6 +400,27 @@ def change_connection_id(self) -> None:
                 "Switching to CID %s (%d)", dump_cid(self._peer_cid), self._peer_cid_seq
             )
 
+    def reset_stream(
+        self,
+        stream_id: int,
+        error_code: int = QuicErrorCode.NO_ERROR,
+    ) -> None:
+        """
+        Reset a stream.
+
+        :param stream_id: The ID of the stream to reset.
+        :param error_code: (optional) An error code indicating why the connection is
+                           being reset. Default: `QuicErrorCode.NO_ERROR`
+        """
+        stream = self._streams[stream_id] = QuicStream(stream_id=stream_id)
```

Comment:
I don't think this is right, we don't want to *create* a new stream. We want to reset an existing stream, and asking for a reset on a stream which doesn't exist should raise an exception.